### ISSUE TITLE
Version v0.3.0

### ISSUE THREAD

**Issue by mahnouman** [tamiat/tamiat#127]
Hello,
This topic is for planning the next release of the CMS, which would be v0.3.0. In this release we aim to provide a minimal set of stable features that we assume would be helpful to our users, whom currently are Vue community developers.

So, the challenge here is to keep it minimal but stable and useful set of features. Feel free to express your ideas and opinions, any and all contributions would be useful.

Thank you

**Comment by mahnouman** [comment]
@mornir @corysimmons  @kieranwild  As an early adopters, Your inputs are most welcomed. Thank you

**Comment by Abdelaziz18003** [comment]
I think we should re-order the project folder structure and have a Nuxt.js like structure. the most important things in my mind are creating a `src/pages` folder .. in it we will have page level components like `home`, `admin`, `posts`, `settings` ... etc. We leave `components` just for partial components so we can break a given page to multiple components. I think the project folder structure now is a bit confusing

**Comment by Pajicc** [comment]
I agree with @Abdelaziz18003 We should make a more understandable structure of folders in src (i think moving pages into pages folder would do the job)
Also maybe we should consider defining all scss we have in seperate files and importing them into the project. That way we could make some styles reusable and have a lot cleaner code

**Comment by gresalahi** [comment]
I think we should also consider  on working in the dashboard by refreshing the UI. Since right know it is okay but we can make it look more neat.

**Comment by corysimmons** [comment]
why are pages and posts a thing?
don't make it wordpress. just make one thing called `data` or something

**Comment by mahnouman** [comment]
@corysimmons I thought of making it "Content" before, so the whole thing is more oriented toward developer point of view.

**Comment by corysimmons** [comment]
Content is fine.
